pagopa/pn-frontend
# Feat/pn 5470

## Short description
Page to view the delegations assigned to a PG

## List of changes proposed in this pull request
- Added DelegationsOfTheCompany page and all components needed

## How to test
Log-in with a PF or a PG and delegate another PG. Log in with that PG and check that page is show correctly, and try to filter out delegations list
